userproject/PotjansMicrocircuit_project/model/runner_guid.txt

```diff
@@ -0,0 +1 @@
+62182A2D-DE80-4612-BF02-C3FF910EF67B 
```

Comment:
What is this?

Reply:
Magic that means GeNN will re-generate a visual studio project with the magic number as the solution